src/git/utils.rs

```diff
@@ -43,7 +43,7 @@ pub fn clone_git_template_into_temp(
     let branch = get_branch_name_repo(&repo)?;
 
     // change from git repository into normal folder.
-    remove_history(git_clone_dir.path())?;
+    // remove_history(git_clone_dir.path())?;
```

Comment:
could you please remove this code here, and also the now unused function?

Reply:
done!